ua_parser/user_agent_parser.py

```diff
@@ -61,15 +61,15 @@ def Parse(self, user_agent_string):
             if self.v1_replacement:
                 v1 = self.v1_replacement
             elif match.lastindex and match.lastindex >= 2:
-                v1 = match.group(2)
+                v1 = match.group(2) or None
```

Comment:
I am curious, is there any impact of this or None? seems like it would just cause `v1 = None` instead of `v1 = ''` - is that the intent?

Reply:
Yes, it was intended to match with the tests in uap-core, e.g. [uap-core/tests/test_os.yaml](https://github.com/ua-parser/uap-core/blob/master/tests/test_os.yaml), where some fields have `None` instead of empty string.